test/e2e/tests/k8s_suite/k8s_suite_test.go

```diff
@@ -201,8 +201,7 @@ func (s *k8sSuite) TestGenericK8s() {
 			utils.VerifyAgentPods(s.T(), c, common.NamespaceName, s.Env().KubernetesCluster.Client(), common.NodeAgentSelector+",agent.datadoghq.com/name=dda-autodiscovery")
 
 			// check agent pods for http check
-			agentPods, err := s.Env().KubernetesCluster.Client().CoreV1().Pods(common.NamespaceName).List(context.TODO(), metav1.ListOptions{LabelSelector: common.NodeAgentSelector + ",agent.datadoghq.com/name=dda-autodiscovery",
-				FieldSelector: "status.phase=Running"})
```

Comment:
Removed `status.phase=Running` as `utils.VerifyAgentPods` already wait for pods to be running